render/detailed/summary.go

```diff
@@ -258,13 +258,24 @@ func podNodeSummary(base NodeSummary, n report.Node) (NodeSummary, bool) {
 	return base, true
 }
 
+var podGroupNodeTypeName = map[string]string{
+	report.Deployment: "Deployment",
+	report.DaemonSet:  "Daemon Set",
+	report.ReplicaSet: "Replica Set",
```

Comment:
yep, good catch. no need for the special minor label if it's not going to share a topo with other types